source/main/gameplay/BeamEngine.cpp

```diff
@@ -248,7 +291,7 @@ void BeamEngine::update(float dt, int doUpdate)
 	if (automode < MANUAL)
 	{
 		// auto-shift
-		if (shifting)
+		if (shifting && !is_Electric) //No shifting in electric cars

```

Comment:
As far as I know, other than E-formula 1, no electric car has gearboxes. But well, i will fix this.